### DISCUSSION TITLE
How to make changes on schema choosing the column position

### DISCUSSION THREAD

**Discussion by ibrahim-brumate** [prisma/prisma#6466] [Q&A]
How can I make changes on schema in my models for use in MySQL with order?
On Sequelize we can set one column after other. How can I do this with prisma?

Ex:
```
ADD COLUMN new_column_name VARCHAR(15) AFTER existing_column_name;
```

Thank you for all support

**Comment by janpio** [comment]
This is not supported right now out of the box, but you could always manipulate the generated migration when adding a new field.

**Reply by ibrahim-brumate** [reply]
thanks for explanation

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
